src/ui/DynamicFacet/DynamicFacet.ts

```diff
@@ -195,14 +197,33 @@ export class DynamicFacet extends Component {
      * See [Normalizing Facet Value Captions](https://developers.coveo.com/x/jBsvAg).
      *
      */
-    valueCaption: ComponentOptions.buildJsonOption<IStringMap<string>>()
+    valueCaption: ComponentOptions.buildJsonOption<IStringMap<string>>(),
+
+    /**
```

Comment:
Suggestion:

> Whether the facet should remain in its current position in the viewport when the mouse cursor is over it.
>
> Leaving this to `true` ensures that the facet does not move around in the search interface while the end-user is interacting with it.
>
> **Default:** `true`

While the rest of the information in the current certainly is interesting, I think it's a bit too technical for reference documentation. If someone wants to know how the option is implemented, they can look at the source code.